### PR TITLE
docs: update dev guide accounting for monrepo structure

### PR DESCRIPTION
**Pull request by juliusgb**

### CODE REVIEW

File: guides/development.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-From the root folder:
+After cloning the repository, from the root folder:
 
-- Run `yarn` to install dependencies.
+- Run `yarn install` to install dependencies.
 
-- Run `yarn start` to start the development server for the package and for the example.
+- Run `yarn build:packages`
+
+- Run `yarn start:www` to start the development server for the package and for the example.
```
Comment (judicaelandria) [inline guides/development.md:9]: actually this only start the website in apps/www
yarn start will start the packages and you can navigate to localhost:5420 to see it

File: guides/development.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
 - Open `localhost:5420` to view the example project.
```
Comment (judicaelandria) [inline guides/development.md:11]: so this would be localhost:3000

**Comment by juliusgb** [comment]
Thanks for the review.

If I understand correctly, my changes make the guide incorrect :sweat_smile:

I've changed it back and added the build step.

**Comment by judicaelandria** [comment]
> Thanks for the review.
> If I understand correctly, my changes make the guide incorrect :person_facepalming:
not really! you just need to add yarn start:www for starting the web app at apps/www
and yarn start to start the whole  app (develop, embedded, etc)
yarn build:packages is fine

**Comment by judicaelandria** [comment]
> > Thanks for the review.
> > If I understand correctly, my changes make the guide incorrect :person_facepalming:

not really! you just need to add yarn start:www for starting the web app at apps/www
and yarn start to start the whole  app (develop, embedded, etc) yarn build:packages is fine

**Comment by steveruizok** [comment]
I've added a postinstall script that should make the manual `yarn build` step unnecessary—and save us a ton of time answering that question on the discord ;)

**Comment by steveruizok** [comment]
If that was the only change, then I'm going to close this one!

**Pull request closed without merging**